scripts/py/dramsim3_utils.py

```diff
@@ -0,0 +1,113 @@
+import sys
+import pandas as pd
+import numpy as np
+
+def fix_dramsim3_json(filename_i, filename_o):
+    jsonfile_o = open(filename_o, "w")
+    with open(filename_i, "r") as jsonfile:
+        lines = [line for line in jsonfile]
+
+    for (lno, line) in enumerate(lines):
+        # if this is the last line
+        # replace the ',' with ']'
+        if lno == len(lines)-1:
+            line = line.replace(',\n',']\n')
+        # write it out
+        jsonfile_o.write(line)
```

Comment:
Little confused why this is necessary

I thought we had a fix that closed out the json file. Is this the same issue?

Reply:
I was surprised too, looks like it didn't make it in to the .tag.json.

Reply:
@marcusnchow Do you remember anything about this? I thought we fixed it.

Reply:
[This](https://github.com/bsg-external/DRAMsim3/blob/1b8828cd293077e321a028310a8b938031b21ff5/src/dram_system.cc#L73) was the fix that i made that replaces the last ']' with a ','

Not sure if this is the same file that this script is working with?
Or do you know if BaseDRAMSystem::PrintStats() is being called?

Reply:
OK, my bad. My helper library uses a smart pointer and it calls DPI finish functions for the device after the finally blocks in the SV. I've added calls to explicitly perform HB cleanup code.

Reply:
Cool